### DISCUSSION TITLE
发布委托:零号空洞捉人

### DISCUSSION THREAD

**Discussion by jis1234** [share121/inter-knot#1740] [其他]
最近我总觉得妻子对我不上心，跟踪之后发现她竟然每天都去零号空洞与以骸深入交流！我决定明早去零号空洞抓他们！
有意愿的绳匠联系我！
奖励:
           抓到女:20菲林
           抓到以骸:80菲林
           全部抓包:90菲林

**Comment by lllliufeng** [comment]
细说深度交流
这个委托我接了

**Reply by jis1234** [reply]
好，明早哨站内部见！

**Reply by robinfanff** [reply]
😁

**Comment by robinfanff** [comment]
zjw同学啥时候有老婆了

**Comment by fu-fu404** [comment]
放开那个以骇，让我来

**Reply by jis1234** [reply]
你配得上200菲林👍🏻

**Reply by lllliufeng** [reply]
有了
